src/Backends/Helmholtz/HelmholtzEOSMixtureBackend.cpp

```diff
@@ -203,8 +199,8 @@ void HelmholtzEOSMixtureBackend::set_binary_interaction_double(const std::string
 
     // Update the values in this instance and the saturation states too
     this->set_mixture_parameters();
-    if (this->SatL){ this->SatL->set_mixture_parameters(); }
```

Comment:
I actually prefer having the braces - it's entirely personal style, but that way if you add to the  actions taken in the true branch, you can be sure you have the correct scope.